csrankings-l.csv

```diff
@@ -836,6 +836,7 @@ Luigia Aiello,Sapienza University of Rome,http://www.diag.uniroma1.it/users/luig
 Luigia Carlucci Aiello,Sapienza University of Rome,http://www.diag.uniroma1.it/users/luigia%20carlucci-aiello,NOSCHOLARPAGE
 Luis A. F. M. Ferreira,Universidade de Lisboa,http://ciisa.fmv.ulisboa.pt/en/research-labs/research-lab-10/pessoas/item/64-luis-ferreira,7eEExREAAAAJ
 Luis Antonio Brasil Kowada,UFF,http://www.professores.uff.br/kowada,8DN4mmEAAAAJ
+Luis Antonio Garcia,University of Utah,https://lagarcia.us,F6Gzg9gAAAAJ

```

Comment:
This is being flagged as a disambiguation entry.
However, this is an alias for Luis Garcia 0001 (https://dblp.org/pid/67/136-1.html)